### PR TITLE
Add support for Contour Plus Blue (UPLOAD-784)

### PR DESCRIPTION
**Pull request by gniezen**
Addresses [UPLOAD-784].

[UPLOAD-784]: https://tidepool.atlassian.net/browse/UPLOAD-784?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by krystophv — DISMISSED** [review]
LGTM 👍🏼

**Comment by gniezen** [comment]
@krystophv Just realised I need another review here before I merge this.

**Review by krystophv — APPROVED** [review]
Updates LGTM 🚀

**Pull request merged**
